src/components/Button.tsx

```diff
@@ -4,14 +4,16 @@ import { Button } from "@mui/material";
 interface ButtonProps {
   label: string;
   onClick: () => void;
+  isDisabled: boolean;
 }
 
-const CustomButton: React.FC<ButtonProps> = ({ label, onClick }) => {
+const CustomButton: React.FC<ButtonProps> = ({ label, onClick, isDisabled }) => {
   return (
     <Button
+      disabled={isDisabled}
       variant="contained"
       style={{
-        backgroundColor: "rgb(67, 102, 134)",
+        backgroundColor: isDisabled ? "grey" : "rgb(67, 102, 134)",
```

Comment:
Instead of grey, consider using a more descriptive name that indicates why the color is changing (e.g., disabledColor).

Reply:
"grey" is a property color from React, like "black" or "red". They are constant and replace manually placed colors with rgb()